Jenkinsfile

```diff
@@ -653,12 +653,12 @@ pipeline {
         stage("Full tests I"){
             when { expression { params.BUILD_CURRENT_STAGE } }
             parallel{
-                stage('Fp32 OpenCL Debug + Codecov') {
+                stage('Fp32 OpenCL Debug (Codecov OFF)') {
                     agent{ label rocmnode("vega") }
                     steps{
                         script{
                             try{
-                                buildHipClangJob('g++', flags: '-DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug', codecov: true)
+                                buildHipClangJob('g++', flags: '-DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug', codecov: false)
```

Comment:
We have same stage in "Smoke Fp32", so disable it.

Reply:
done